cli/cmd/install/install_test.go

```diff
@@ -187,3 +187,39 @@ func getInitializedCommand(t *testing.T) *Command {
 	c.init()
 	return c
 }
+
+func TestCheckValidEnterprise(t *testing.T) {
+	c := getInitializedCommand(t)
+	c.kubernetes = fake.NewSimpleClientset()
+	secret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "consul-secret",
+		},
+	}
+	secret2 := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "consul-secret2",
+		},
+	}
+
+	// Enterprise secret and image are valid.
+	c.kubernetes.CoreV1().Secrets("consul").Create(context.Background(), secret, metav1.CreateOptions{})
+	c.kubernetes.CoreV1().Secrets("unrelated").Create(context.Background(), secret, metav1.CreateOptions{})
```

Comment:
```suggestion
	c.kubernetes.CoreV1().Secrets("unrelated").Create(context.Background(), secret2, metav1.CreateOptions{})
```

I think this is supposed to be secret2 for the test case in L221